website/views.py

```diff
@@ -5,23 +5,55 @@
 
 
 def page_home(request):
-    return render(request, "page_home.html")
+    schedule = [
+        [
+            [('Jan 3rd', 'Announce Conference', True)],
+            [('Jul 7th', 'Conference Schedule is announced', False)],
+        ],
```

Comment:
Hey @dduafa,

On the schedules list, we have to manually set whether the schedule deadline has passed to apply the `line-through`, which can be difficult to manage, I would suggest the following:

Create a model to manage the schedules.

or

Instead of just passing `True` or `False`, compare the schedule deadline to a future date, like so:

```
datetime.now() > datetime(2025, 1, 3)
```